src/@next/components/molecules/AccountMenuMobile/stories.tsx

```diff
@@ -23,10 +24,14 @@ const DEFAULT_PROPS = { ...{ links, active } };
 storiesOf("@components/molecules/AccountMenuMobile", module)
   .addParameters({ component: AccountMenuMobile })
   .addDecorator(story => (
-    <MemoryRouter initialEntries={["/"]}>{story()}</MemoryRouter>
+    <IntlProvider locale="en">
+      <MemoryRouter initialEntries={["/"]}>{story()}</MemoryRouter>
+    </IntlProvider>
   ))
   .add("default", () => (
-    <Wrapper>
-      <AccountMenuMobile {...DEFAULT_PROPS} />
-    </Wrapper>
+    <IntlProvider locale="en">
```

Comment:
Does the story require `IntlProvider` if it is already added in the decorator?

Reply:
Of course not. Thank you for your vigilance @orzechdev. Found the same mistake in `AccountMenu`. Sorry for the oversight. Pushing fix